tokio-stream/Cargo.toml

```diff
@@ -37,6 +37,8 @@ async-stream = "0.3"
 parking_lot = "0.12.0"
 tokio-test = { path = "../tokio-test" }
 futures = { version = "0.3", default-features = false }
+futures-test = "0.3.26"
+futures-util = "0.3.26"
```

Comment:
I don't think we need these. We already import `futures`.

Reply:
Note that `FutureExt` is available from the `futures` crate.

Reply:
I removed `futures-test`. Unfortunately I still need `futures-util` to import `FutureExt` as I needed to get `now_or_never()` to work.

I noticed that `tokio/src/sync/watch.rs` is using `recv.changed().now_or_never()` from `crate::sync::watch::channel(0i32)` in some tests. I would like to investigate this a little further. Any pointers would still be highly appreciated as before.

Reply:
`futures::future::FutureExt` worked for me thanks!